projects/admin/src/app/circulation/patron/loan/loan.component.ts

```diff
@@ -171,11 +178,18 @@ export class LoanComponent implements OnInit {
         if (err && err.error && err.error.status) {
           errorMessage = err.error.status;
         }
-        this.toastService.error(
-          this.translate.instant('An error occured on the server: ') +
-            errorMessage,
-          this.translate.instant('Checkin')
-        );
+        if ( err.error.status.includes('403 Forbidden')) {
+          this.toastService.error(
+            this.translate.instant('Checkout is not allowed by circulation policy'),
+            this.translate.instant('Checkout')
+          );
+        } else {
+          this.toastService.error(
+            this.translate.instant('An error occured on the server: ') +
```

Comment:
`this.translate.instant('An error occured on the server: {{ errorMessage }} ', { errorMessage })`